lms/templates/base.html.jinja2

```diff
@@ -32,7 +32,19 @@
       {% block content %}{% endblock %}
     </div>
 
-    <script type="application/json" class="js-config">{{ js_config|tojson }}</script>
-    {% block scripts %}{% endblock %}
+    {% block scripts %}
+      <script type="application/json" class="js-config">{{ js_config|tojson }}</script>
+
+      {% if context.rpc_server_config %}
+        <script type="application/json" class="js-rpc-server-config">{{ context.rpc_server_config|tojson }}</script>
+        {% for url in asset_urls("postmessage_json_rpc_server_js") %}
+          <script src="{{ url }}"></script>
+        {% endfor %}
+      {% endif %}
+
+      {% if context.hypothesis_config %}
+        <script type="application/json" class="js-hypothesis-config">{{ context.hypothesis_config|tojson }}</script>
```

Comment:
Hmm. The new file picker frontend uses the same `js-hypothesis-config` class for its config block which is confusing. Probably better to:

1) Use a more explicit name for the Hypothesis client config (`js-hypothesis-client-config`?)
2) Change the LMS frontend to use a different name for its config blog (`js-config`)?

Reply:
Yep, the plan is to change the new file picker to read js-config, yes. Note that js-hypothesis-config is _already_ used by the backend and the JavaScript RPC server to contain the Hypothesis client config, so the file picker templates that're rendering a second `js-hypothesis-config` are in conflict with that. That presumably would break the RPC server -> client integration but as it happens there are no pages that contain both the file picker and the client together, so nothing was broken.

Regarding the naming of the config objects, I was thinking in future to merge them into a single js-config object like this:

```
<script type="application/json" class="js-hypothesis-config">
    {
        // Top-level JS settings (e.g. for the file picker) go here.
        ...
        "rpc_server": { ... },
        "hypothesis_client": { ... },
</script>
```

The file-picker's settings could potentially go under their own `file_picker` key as well.